### ISSUE TITLE
dep broken because of moved fsnotify package

### ISSUE THREAD

**Issue by aaron7** [weaveworks/cortex#762]
```
Solving failure: No versions of gopkg.in/fsnotify.v1 met constraints:
        v1.4.2: unable to update checked out version: fatal: reference is not a tree: 629574ca2a5df945712d3079857300b5e4da0236
: command failed: [git checkout 629574ca2a5df945712d3079857300b5e4da0236]: exit status 128
make: *** [update-vendor] Error 1
```

Merged https://github.com/prometheus/prometheus/pull/3995 to fix upstream.

Will now update in cortex.

**Comment by aaron7** [comment]
Reverted #763

**Comment by bboreham** [comment]
Closed by #802
